Assets/Scripts/MainMenu.cs

```diff
@@ -9,4 +9,15 @@ public void LoadMainGame()
     {
         SceneManager.LoadScene(1);
     }
+
+    public void LoadSettingsMenu()
```

Comment:
so we should make an enum for something like this so that we aren't using hardcoded numbers throughout the codebase. the issue with using hardcoded numbers like this is that to another programmer they look completely random (i have no idea what `1` means or points to here and it would require additional digging on my part to find out, which is not ideal and quickly makes other programmers mad at you lol).

so you could handle this multiple ways. you could use an [enum](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum) like i suggested above, where it would something like

```
enum GameScenes {
  Main,
  Settings,
  [... etc]
}
```
and you would reference it here like

`SceneManager.LoadScene(GameScene.Main);`

you could also use [constants](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants). so you could do something like this:
```
private const int MainScene = 1;
```

and then reference it by doing

`SceneManager.LoadScene(MainScene);`

Note: i tend to prefer going the enum route, i find it to be cleaner in this kind of context.